### DISCUSSION TITLE
CANCEL GESTURE

### DISCUSSION THREAD

**Discussion by cho66x** [naverz/zepeto-studio-global#49] [Q&A]
how to undo motion in animation clip?

**Comment by jhslee** [comment]
Hello @anggacho 👋 

Please cache your `ZepetoCharacter` instance, then use ZepetoCharacter::CancelGesture, to cancel the animation clip. 

i.e. 


```
//Spawn and cache the zepeto character instance. 
ZepetoPlayers.instance.OnAddedLocalPlayer.AddListener(() => {
            this.zepetoCharacter = ZepetoPlayers.instance.LocalPlayer.zepetoPlayer.character;
        });
        

//Play the Gesture.
this.zepetoCharacter.SetGesture(this.animationClip);

[Later...]

//Cancel Gesture
this.zepetoCharacter.CancelGesture();
```


I encourage you to check out an example project on this to see this working for yourself! 
https://github.com/naverz/zepeto-world-sample/tree/main/Assets/Chapter3

**Comment by cho66x** [comment]
Thank U, for your very helpful answer
